README.rst

```diff
@@ -94,6 +94,19 @@ Volto Blocks Support
 plone.volto enables the new Volto Blocks editor on ``Document``, ``Language Root Folder`` and ``Site Root``.
 
 
+Block Types Index
+-----------------
+
+plone.volto adds a ``block_types`` index to the Plone catalog.
+It can be used to query for items that use a particular type of block::
+
+  portal_catalog.searchResults(block_types="image")
+
+The ``block_types`` index was added in plone.volto 4.1.0.
```

Comment:
If it makes sense, maybe move this directly beneath the subheading?

```suggestion
.. versionadded:: 4.1.0
```

Reply:
@stevepiercy This directive doesn't appear to be rendered well by github, so I'm going to put this back to how it was.

<img width="497" alt="Screenshot 2024-01-07 at 7 33 23 PM" src="https://github.com/plone/plone.volto/assets/49014/3be88425-a4c1-4300-8219-19b58c3f55cf">

Reply:
Yeah, it's reStructuredText support is pretty weak. Markdown has better support.